### PR TITLE
US 1464 change removebinding feature

### PR DESCRIPTION
**Pull request by NicolasPacqueriaud**
Updating NodeDto when a binding is active. 
Updating bindings behaviors. The binding methods return now SetEntityProperties. Adding comments in both scripts.

### CODE REVIEW

File: UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs [hunk 1]
```diff
@@ -177,11 +188,16 @@ Transform InspectBoundRigs(BoneBindingDto dto)
...
         IEnumerator WaitForRig(BoneBindingDto dto, UMI3DClientUserTrackingBone bone)
         {
-            UMI3DNodeInstance node = UMI3DEnvironmentLoader.GetNode(dto.objectId);
+            UMI3DNodeInstance node;
+            var wait = new WaitForFixedUpdate();
+
+            while ((node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null)
+            {
+                yield return wait;
+            }
```
Comment (Julien-Casarin) [inline UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs:197]: Prefer Unity's syntax: 
`yield return new WaitWhile(() => (node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null )`

File: UMI3D-SDK/Assets/ClientDevelopmentKit/Core/Runtime/Loaders/UMI3DNodeLoader.cs [hunk 2]
```diff
@@ -71,7 +71,22 @@ public override bool SetUMI3DProperty(UMI3DEntityInstance entity, SetEntityPrope
...
+                switch (property.property)
+                {
+                    case UMI3DPropertyKeys.Position:
+                        gltfDto.position = (SerializableVector3)property.value;
+                        break;
+                    case UMI3DPropertyKeys.Rotation:
+                        gltfDto.rotation = (SerializableVector4)property.value;
+                        break;
+                    default:
+                        break;
```
Comment (Schlappinou) [inline UMI3D-SDK/Assets/ClientDevelopmentKit/Core/Runtime/Loaders/UMI3DNodeLoader.cs:86]: don't this need to do something in with scale ?

File: UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs (hunk 1 above)
Comment (Julien-Casarin) [reply UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs:197]: What is the usage of wait ?

File: UMI3D-SDK/Assets/ClientDevelopmentKit/Core/Runtime/Loaders/UMI3DNodeLoader.cs (hunk 2 above)
Comment (TheoDesbonnets) [inline UMI3D-SDK/Assets/ClientDevelopmentKit/Core/Runtime/Loaders/UMI3DNodeLoader.cs:86]: why don't you set scale ?

File: UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs (hunk 1 above)
Comment (TheoDesbonnets) [reply UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs:197]: Note that `WaitWhile` will be evaluate each frame whereas `WaitForFixedUpdate` will skip some.

File: UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs [hunk 3]
```diff
@@ -177,11 +188,16 @@ Transform InspectBoundRigs(BoneBindingDto dto)
 
         IEnumerator WaitForRig(BoneBindingDto dto, UMI3DClientUserTrackingBone bone)
         {
-            UMI3DNodeInstance node = UMI3DEnvironmentLoader.GetNode(dto.objectId);
+            UMI3DNodeInstance node;
+            var wait = new WaitForFixedUpdate();
+
+            while ((node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null)
```
Comment (Julien-Casarin) [inline UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs:194]: `yield return new WaitWhile(() => (node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null )`

File: UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs [hunk 4]
```diff
@@ -177,11 +188,16 @@ Transform InspectBoundRigs(BoneBindingDto dto)
 
         IEnumerator WaitForRig(BoneBindingDto dto, UMI3DClientUserTrackingBone bone)
         {
-            UMI3DNodeInstance node = UMI3DEnvironmentLoader.GetNode(dto.objectId);
+            UMI3DNodeInstance node;
+            var wait = new WaitForFixedUpdate();
```
Comment (Julien-Casarin) [inline UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs:192]: Where is it used ?

File: UMI3D-SDK/Assets/EnvironmentDevelopmentKit/UserCapture/Runtime/Tracking/UMI3DEmbodimentManager.cs [hunk 5]
```diff
@@ -152,15 +159,25 @@ protected void DeleteEmbodiment(UMI3DUser user)
...
         {
             LoadEntity op = node.Register();
-            transaction.Operations.Add(op);
+            op += UMI3DEnvironment.GetEntities<UMI3DUser>();
+
+            UMI3DServer.Dispatch(new Transaction
+            {
+                Operations = new List<Operation> { op },
+                reliable = true
+            });
```
Comment (TheoDesbonnets) [inline UMI3D-SDK/Assets/EnvironmentDevelopmentKit/UserCapture/Runtime/Tracking/UMI3DEmbodimentManager.cs:174]: the loadEntity return with `LoadEntity op = node.Register();` should already contain 'UMI3DEnvironment.GetEntities<UMI3DUser>();'

**Review by TheoDesbonnets — CHANGES_REQUESTED** [review]

**Review by Julien-Casarin — APPROVED** [review]

**Review by Schlappinou — APPROVED** [review]

**Review by TheoDesbonnets — APPROVED** [review]

**Pull request merged**